src/components/cohortStudent/index.js

```diff
@@ -0,0 +1,71 @@
+import { get } from '../../service/apiClient'
+import { useState, useEffect } from 'react'
+import Card from '../card'
+import CohortTitleCard from '../CohortTitleCard'
+import StudentCard from '../studentCard'
+import './style.css'
+import TeacherCard from '../TeacherCard'
+
+const CohortStudent = () => {
+
+    const [cohortInfo, setCohortInfo] = useState(null)
+    const [cohortStudents, setCohortStudents] = useState(null)
+    const [teachers, setTeachers] = useState(null)
+
+
+    useEffect(() => {
+        const getCohortInfo = async () => {
+            const res = await get(`cohorts/1`)
+            setCohortInfo(res.data.cohort)
+        }
+        getCohortInfo()
+
+        const getCohortMembers = async () => {
+            const res = await get(`users`)
+            const students = res.data.users.filter((user)=> user.role==='STUDENT')
+            const teachers = res.data.users.filter((user)=> user.role==='TEACHER')
+            setCohortStudents(students)
+            setTeachers(teachers)
+        }
+        getCohortMembers()
+
+    }, [])
+
+    return (
+        <>
+            <main>
+                <Card >
+                    <h3 className="text-blue border-bottom"> My cohort </h3>
+                    {cohortInfo && <CohortTitleCard info={cohortInfo}/>}
+                    <ul className='studentList'>
+                        {cohortStudents?.map((person, index) => {
```

Comment:
Instead of doing a null check for `cohortStudents`, you should always initialise array state as an empty array:

```js
const [cohortStudents, setCohortStudents] = useState([])
```

Then you can safely do the map without a null check
```js
{cohortStudents.map((person, index) => {
```

Reply:
Good to know! We will make the changes now.

Reply:
This has not been fixed as well